Materials/Materials-04-Introduction_to_Valtools/Materials-04-E01-Validation_Test_Code/Materials-04-E01-verbose.R

```diff
@@ -0,0 +1,58 @@
+########### Task 04_E01 A
+# render the validation report found in folder "simple_validation"
```

Comment:
in the "verbose" we might want to provide the setwd

At the end of the project, should we have them set the wd back to the root of the tutorial?